docs/installation.rst

```diff
@@ -107,8 +108,10 @@ MTF supports Gherkin-based testing in Python. To write tests in a natural langua
     $ sudo pip install behave
 
 .. _installing_mtg:
+
 Installing MTF
-~~~~~~~~~~~~~
+~~~~~~~~~~~~~~
+
 Install MTF rpm from `Fedora Copr repo`_.
 
 .. _Fedora Copr repo: https://copr.fedorainfracloud.org/coprs/phracek/Modularity-testing-framework/
```

Comment:
The same here: https://copr.fedorainfracloud.org/coprs/phracek/meta-test-family/